src/entities/purchase-result.ts

```diff
@@ -19,4 +19,15 @@ export interface PurchaseResult {
    * The operation session id of the purchase.
    */
   readonly operationSessionId: string;
+
+  /**
+   * The email of the customer who made the purchase.
+   */
+  readonly customerEmail: string | null;
+
+  /**
+   * The date when the subscription will renew.
+   * This will be null for non-subscription purchases or lifetime subscriptions.
+   */
+  readonly renewalDate: Date | null;
```

Comment:
Wouldn't this be accessible by the `customerInfo` property already? While it's true that you would need to know the package/product just purchased, the client should be aware of that already right? Just thinking it seems like it's duplicating information...